lms/djangoapps/django_comment_client/management/management_utils.py

```diff
@@ -0,0 +1,49 @@
+'''
+This module provides utility function for the utility functions for django_comment_client
+'''
+from django.contrib.auth.models import User
+from pymongo import MongoClient
+import logging
+from django.db import IntegrityError
+from pymongo.errors import PyMongoError
+
+log = logging.getLogger(__name__)
+
+
+def rename_user(old_username, new_username):
+    '''
+    This function changes the username of a user
+    :param old_username: old username of the user
+    :param new_username: new username desired for this user
+    :return: result {'status': 'success'/'failure', 'message': 'relevant message'}
+    '''
+    result = {'status': 'failure', 'message': ''}
+    try:
+        user = User.objects.get(username=old_username)
+    except User.DoesNotExist:
+        result['message'] = "User %s does not exist." % old_username
+        log.exception(result['message'])
+        return result
+    try:
+        user.username = new_username
+        user.save()
+    except IntegrityError:
+        result['message'] = 'Error modifying the username for %s. Rename not performed.' % old_username
+        log.exception(result['message'])
+        return result
+    else:
```

Comment:
Since we're going to return above in the `except` block, the `else`
block is a superfluous level of indentation. We can remove the `else`
and unindent the contents.